app/controllers/assignments_controller.rb

```diff
@@ -105,6 +107,18 @@ def update
     end
   end
 
+  def feed
+    user = (@current_user || User.find_by(calendar_access_token:
```

Comment:
Won't `@current_user` always be `nil`? Since you skipped the before action that sets it?